### PR TITLE
Update drutiny support to 2.3.*@dev

### LINKED ISSUE #2: Recommended usage does not work.
Running the prescribed command to establish a new codebase for development simply does not work.

```
 $ composer create-project --no-interaction --prefer-source -s dev drutiny/project-dev drutiny-dev
    1/12:       http://repo.packagist.org/p/provider-archived$810acf97b71c6fba28f8543dd40e53f907f9a33f175e2633cd2c9b88aa25bd17.json
    2/12:       http://repo.packagist.org/p/provider-latest$cf794f5757c59810e599b132d4c3bef5c4c45a5a57bddb367739f77438d71b0e.json
    3/12:       http://repo.packagist.org/p/provider-2018-04$127cc7238dc10c80b9fcdfaa474a28d17a7d496b1332289c8d0cfa924ada1d45.json
    4/12:       http://repo.packagist.org/p/provider-2019-01$2657ac9494b2e3c5aea8d915dcb6ffe0738e679310c305714e1105d997d2cfb8.json
    5/12:       http://repo.packagist.org/p/provider-2014$5b6e5153362c83bc4e59cc5673757499301eeb3d73e5409bc140c67d7bc6098f.json
    6/12:       http://repo.packagist.org/p/provider-2013$df0985e9874bab2485ff1bb641a32254a024aee1f18e9b4f2a47f7397a8b3231.json
    7/12:       http://repo.packagist.org/p/provider-2018-07$f4d38ec51763fc750613b9e826c28ba7671941c686cdfe1707295213ee4376df.json
    8/12:       http://repo.packagist.org/p/provider-2018-10$ab639fdd00999096d2c3f8e9c89512e01a494db39a8054e2926a4573c3934245.json
    9/12:       http://repo.packagist.org/p/provider-2018$4b6dabb9ced6ba572acd3a17040f38126c0bd910220a44f7bbef61cf0d74aa2d.json
    10/12:      http://repo.packagist.org/p/provider-2015$f028b8cf6836bb85f09aeecaa51c11cdc74640d8458458d1338f3e662a68ad7a.json
    11/12:      http://repo.packagist.org/p/provider-2016$29c41a7473523a422be1955bfa32da7819f5bc9267bb540c7a946ae941219712.json
    12/12:      http://repo.packagist.org/p/provider-2017$0aa5756c81858110bae840319fa5b71f43ee5cb1fdbb43821ced92a255251bbd.json
    Finished: success: 12, skipped: 0, failure: 0, total: 12
    1/12:       http://repo.packagist.org/p/provider-archived$810acf97b71c6fba28f8543dd40e53f907f9a33f175e2633cd2c9b88aa25bd17.json
    2/12:       http://repo.packagist.org/p/provider-latest$cf794f5757c59810e599b132d4c3bef5c4c45a5a57bddb367739f77438d71b0e.json
    3/12:       http://repo.packagist.org/p/provider-2014$5b6e5153362c83bc4e59cc5673757499301eeb3d73e5409bc140c67d7bc6098f.json
    4/12:       http://repo.packagist.org/p/provider-2013$df0985e9874bab2485ff1bb641a32254a024aee1f18e9b4f2a47f7397a8b3231.json
    5/12:       http://repo.packagist.org/p/provider-2019-01$2657ac9494b2e3c5aea8d915dcb6ffe0738e679310c305714e1105d997d2cfb8.json
    6/12:       http://repo.packagist.org/p/provider-2018-04$127cc7238dc10c80b9fcdfaa474a28d17a7d496b1332289c8d0cfa924ada1d45.json
    7/12:       http://repo.packagist.org/p/provider-2018-10$ab639fdd00999096d2c3f8e9c89512e01a494db39a8054e2926a4573c3934245.json
    8/12:       http://repo.packagist.org/p/provider-2018$4b6dabb9ced6ba572acd3a17040f38126c0bd910220a44f7bbef61cf0d74aa2d.json
    9/12:       http://repo.packagist.org/p/provider-2018-07$f4d38ec51763fc750613b9e826c28ba7671941c686cdfe1707295213ee4376df.json
    10/12:      http://repo.packagist.org/p/provider-2015$f028b8cf6836bb85f09aeecaa51c11cdc74640d8458458d1338f3e662a68ad7a.json
    11/12:      http://repo.packagist.org/p/provider-2017$0aa5756c81858110bae840319fa5b71f43ee5cb1fdbb43821ced92a255251bbd.json
    12/12:      http://repo.packagist.org/p/provider-2016$29c41a7473523a422be1955bfa32da7819f5bc9267bb540c7a946ae941219712.json
    Finished: success: 12, skipped: 0, failure: 0, total: 12
Installing drutiny/project-dev (2.x-dev c07625c6902e4c98863483324ccb4161151e4497)
  - Installing drutiny/project-dev (2.x-dev c07625c): Cloning c07625c690 from cache
Created project in drutiny-dev
    1/1:        http://repo.packagist.org/p/provider-latest$e445dbb5c2c6852564c5be0b8f371956dd888e47de0e7f02f2ae9c27ef9af0e7.json
    Finished: success: 1, skipped: 0, failure: 0, total: 1
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for drutiny/acquia 2.x-dev -> satisfiable by drutiny/acquia[2.x-dev].
    - drutiny/plugin-distro-common 2.x-dev requires drutiny/drutiny 2.1.*@dev || 2.2.*@dev -> satisfiable by drutiny/drutiny[2.1.x-dev, 2.2.0, 2.2.1, 2.2.1-beta1, 2.2.2, 2.2.x-dev].
    - Can only install one of: drutiny/drutiny[2.3.x-dev, 2.2.0].
    - Can only install one of: drutiny/drutiny[2.3.x-dev, 2.2.1].
    - Can only install one of: drutiny/drutiny[2.3.x-dev, 2.2.1-beta1].
    - Can only install one of: drutiny/drutiny[2.3.x-dev, 2.2.2].
    - Can only install one of: drutiny/drutiny[2.3.x-dev, 2.2.x-dev].
    - Can only install one of: drutiny/drutiny[2.3.x-dev, 2.1.x-dev].
    - drutiny/acquia 2.x-dev requires drutiny/drutiny 2.3.*@dev -> satisfiable by drutiny/drutiny[2.3.x-dev].
    - Installation request for drutiny/plugin-distro-common 2.x-dev -> satisfiable by drutiny/plugin-distro-common[2.x-dev].
```

`drutiny/plugin-distro-common` isn't marked as compatible.
Here's a working diff with the package missing...

```
diff --git a/composer.json b/composer.json
index 3e26957..683ce3d 100644
--- a/composer.json
+++ b/composer.json
@@ -4,11 +4,10 @@
   "description": "Project template for Drutiny development",
   "minimum-stability": "dev",
   "require": {
-    "drutiny/drutiny": "2.2.*@dev",
+    "drutiny/drutiny": "2.3.x-dev",
     "drutiny/acquia": "2.x-dev",
     "drutiny/plugin-drupal-7": "2.x-dev",
     "drutiny/plugin-drupal-8": "2.x-dev",
-    "drutiny/plugin-distro-common": "2.x-dev",
     "drutiny/sumologic": "2.x-dev",
     "drutiny/http": "2.x-dev",
     "drutiny/cloudflare": "2.x-dev"
```

### PR DESCRIPTION
**Pull request by fubarhouse**
Second PR which collectively with https://github.com/drutiny/plugin-distro-common/pull/1 resolves #2

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,11 +4,10 @@
   "description": "Project template for Drutiny development",
   "minimum-stability": "dev",
   "require": {
-    "drutiny/drutiny": "2.2.*@dev",
+    "drutiny/drutiny": "2.3.x-dev",
```
Comment (fiasco) [inline composer.json:7]: This should use `2.3.*@dev` as this allows users to pin to stable releases or run dev (rather than force dev).

**Review by fiasco — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (fubarhouse) [reply composer.json:7]: Changes committed.

**Review by fiasco — APPROVED** [review]

**Comment by fiasco** [comment]
Changes look find by it looks like #4 stuffed up the rebasing on this one. Can you rebase the PR and force push the change?

**Comment by fubarhouse** [comment]
Apologies, the branch has been rebased from upstream/2.x and force-pushed.

**Pull request merged**